src/_pytest/hookspec.py

```diff
@@ -496,27 +496,6 @@ def pytest_terminal_summary(terminalreporter, exitstatus):
     """
 
 
-@hookspec(historic=True)
-def pytest_logwarning(message, code, nodeid, fslocation):
```

Comment:
we should keep this hook in and issue a warning about it not being called anymore, sheduling that removal for later

Reply:
Hmm when do we issue the warning?

Reply:
we warn for implementations of the hook using warn_on_impl of pluggy

Reply:
Oh right. Shame we forgot to do that earlier...

Reply:
Done!